packages/hpe_aruba_cx/data_stream/log/fields/fields.yml

```
@@ -0,0 +1,895 @@
+
+        - name: ft_air_req
+          type: keyword
+          description: >
+
+        - name: ft_dir
+          type: keyword
+          description: >
+
+        - name: ft_num
+          type: long
+          description: >
+
+        - name: function
+          type: keyword
+          description: >
+
+        - name: minimum
+          type: long
+          description: >
+
+        - name: module_idx
+          type: long
+          description: >
+
+        - name: name
+          type: keyword
+          description: >
+
+        - name: new_status
+          type: keyword
+          description: >
+
+        - name: old_status
+          type: keyword
+          description: >
+
+        - name: speed_idx_status
+          type: keyword
+          description: >
+
+        - name: speedval
+          type: long
+          description: >
+
+        - name: tray_index
+          type: long
+          description: >
+
+        - name: air_flow_direction
+          type: keyword
+          description: >
+
+        - name: zone_idx
+          type: long
+          description: >
+
+    - name: firmware
+      type: group
+      fields:
+        - name: dnld_type
+          type: keyword
+          description: >
+
+        - name: image_profile
+          type: keyword
+          description: >
+
+    - name: hardware
+      type: group
+      fields:
+        - name: addr
+          type: keyword
+          description: >
+
+        - name: bus
+          type: keyword
+          description: >
+
+        - name: cap
+          type: keyword
+          description: >
+
+        - name: channel
+          type: keyword
+          description: >
+
+        - name: cpus
+          type: long
+          description: >
+
+        - name: device
+          type: keyword
+          description: >
+
+        - name: function
+          type: keyword
+          description: >
+
+        - name: ip
+          type: ip
+          description: >
+
+        - name: level
+          type: keyword
+          description: >
+
+        - name: location
+          type: keyword
+          description: >
+
+        - name: mcgstatus
+          type: keyword
+          description: >
+
+        - name: misc
+          type: keyword
+          description: >
+
+        - name: offlined
+          type: long
+          description: >
+
+        - name: origin
+          type: keyword
+          description: >
+
+        - name: page
+          type: keyword
+          description: >
+
+        - name: seg
+          type: keyword
+          description: >
+
+        - name: socket
+          type: keyword
+          description: >
+
+        - name: test_name
+          type: keyword
+          description: >
+
+        - name: type
+          type: keyword
+          description: >
+
+        - name: vni
+          type: keyword
+          description: >
+
+    - name: instance
+      type: group
+      fields:
+        - name: id
+          type: long
```

```suggestion
          type: keyword
```

(There's more below, not worth commenting on every single one)